framework/components/ecl-footers/_footers.scss

```diff
@@ -1,142 +1,122 @@
-/*
-Footer
-
-Site-wide footer component
-
-Markup: html/footer.html
-
-Style guide footer
-*/
-
-.footer {
-  background-color: $blue-medium;
+/**
+ * Footers component
+ * @define footers ; weak
+ */
+
+.ecl-footers__site-identity,
+.ecl-footers__site-corporate,
+.ecl-footers__ec {
+  @extend %hidden-print;
 }
 
-.footer-bottom {
-  background-color: $brand-primary;
-  padding: 0.9em 0;
+.ecl-footers__social-icon {
+  &::before {
+    margin-right: 8px !important;
+  }
+
+  &::after {
+    content: none !important;
+    display: none !important;
+  }
 }
 
-.footer,
-.footer-bottom {
-  @extend %hidden-print;
-  color: $white;
+.ecl-footers__title {
+  border-bottom: 2px solid map-get($colors, 'blue-50');
+  color: #fff;
+  margin-bottom: 1.1rem;
+  padding-bottom: 0.5rem;
 
-  a {
-    color: $white;
-    text-decoration: underline;
-    &:hover,
-    &:active,
-    &:focus {
-      text-decoration: none;
-    }
-  }
-  // Inline listings need more padding-right on the footer.
-  .list-inline {
-    .footer__menu-item {
-      padding-right: 15px;
-    }
+  @include media-breakpoint-between(md, lg) {
+    // Make titles same height on mobile breakpoint.
+    min-height: 3.2rem;
   }
-}
 
-.footer__label {
-  display: block;
-  margin-right: 20px;
+  // Make titles height auto on large screens.
+  @media (min-width: 1291px) {
```

Comment:
Another very specific rule in the codebase of europa-theme i'm afraid to touch to avoid broken hearts and big conflicts.